cupy/random/generator.py

```diff
@@ -739,7 +740,9 @@ def seed(self, seed=None):
                     numpy.asarray(seed).astype(numpy.uint64, casting='safe'))
 
         curand.setPseudoRandomGeneratorSeed(self._generator, seed)
-        curand.setGeneratorOffset(self._generator, 0)
+        if(self.method != curand.CURAND_RNG_PSEUDO_MT19937 and
+           self.method != curand.CURAND_RNG_PSEUDO_MTGP32):
```

Comment:
nit here:
```python
if (self.method not in (curand.CURAND_RNG_PSEUDO_MT19937, 
        curand.CURAND_RNG_PSEUDO_MTGP32):
```

Reply:
Done. Should I also add a test case as there wasn't one that tests all the different methods available?

Reply:
would be great to have